extensions/linz/examples/collection.json

```diff
@@ -0,0 +1,36 @@
+{
+  "stac_version": "1.0.0",
+  "stac_extensions": [
+    "https://stac-extensions.github.io/item-assets/v1.0.0/schema.json",
+    "https://stac-extensions.github.io/template/v1.0.0/schema.json"
```

Comment:
This should link to our extension so that the file will be verified against our schema.

Reply:
Extensions removed